src/frontend/src/pages/PIAIntakeForm/index.tsx

```diff
@@ -376,6 +429,16 @@ const PIAIntakeFormPage = () => {
           )}
         </form>
       </section>
+      <Modal
+        confirmLabel={modalConfirmLabel}
+        cancelLabel={modalCancelLabel}
+        titleText={modalTitleText}
+        show={showModal}
```

Comment:
since reverse is one of the props, can we have reverse={false}

Reply:
The 'reverse' prop is really only for the Header calling the sign out modal. Because it is an optional property I don't see the necessity for further verbosity. Could we compromise and say that if further implementations use the 'reverse' prop, then we can begin adding it explicitly to every instance?